src/applications/find-va-forms/api/index.js

```diff
@@ -15,5 +18,10 @@ export const fetchFormsApi = async (URL, query) => {
   await new Promise(resolve => setTimeout(resolve, 1000));
 
   // Give back the normalized forms data.
-  return forms;
+  const normalizedForms = normalizeFormsForTable(forms);
+
+  // Sort the forms by 'id' and 'asc' by default.
+  const sortedForms = orderBy(normalizedForms, 'id', 'asc');
```

Comment:
Would be nice if the future API endpoint would have options for us to receive the forms presorted 🙂